client/src/styles/components/_renku_tree.scss

```
@@ -0,0 +1,68 @@
+.rk-tree-item {
+  text-decoration: none;
+  color: $rk-dark;
+  background-color: white;
+  border: solid 1px $light;
+
+  &:not(:last-child) {
+    border-bottom: none !important;
+  }
+
+  &:hover {
+    background-color: $rk-background-0;
+  }
+  
+  &.selected {
+    background-color: $rk-light !important;
+  }
+
+  &.indentation-1 {
+    margin-left: 1em;
+  }
```

I think this way might be too brittle. How about something like :
```suggestion
  &.indentation {
   --renku-tree-indent: 1;
    margin-left: calc(1em * var(--renku-tree-indent));
  }
```

---

I ended up removing this and optioning for the "element style" solution